test/unit/openshift/cluster.test.ts

```diff
@@ -211,7 +211,7 @@ suite('Openshift/Cluster', function() {
             });
 
             test('checks cluster url name is valid url', async () => {
-                let result: string | Thenable<string>;
+                let result: any;
```

Comment:
Not a huge deal, but feel free to leave out the type if the best description is `any`.

Reply:
Fixed